pkgs/applications/science/biology/platypus/default.nix

```diff
@@ -0,0 +1,38 @@
+{stdenv, fetchFromGitHub, python27, htslib, zlib, makeWrapper}:
+
+let python = python27.withPackages (ps: with ps; [ cython ]);
+
+in stdenv.mkDerivation rec {
+  name = "platypus-${version}";
+  version = "git-2017-03-07";
```

Comment:
This should be `name = "platypus-unstable-2017-03-07";`. See https://nixos.org/nixpkgs/manual/#sec-package-naming.